lib/msc_extensions/msc_3381_polls/poll_room_extension.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:matrix/matrix.dart';
+
+extension PollRoomExtension on Room {
+  static const String mTextJsonKey = 'org.matrix.msc1767.text';
+  static const String startType = 'org.matrix.msc3381.poll.start';
```

Comment:
These are getting duplicated here since we also declare them in `PollEventContent` class